### PR TITLE
Complete and fix native article documentation based on Copilot sample

### PR DESCRIPTION
**Pull request by almostagile**

### CODE REVIEW

File: source/includes/_api.md [hunk 1]
```diff
@@ -67,6 +103,8 @@ that are supported by the integration API.
 
 ### Response
 
+A [paginated response](#pagination) of type `article`, `nativearticle` or both.
```
Comment (thekemkid) [inline source/includes/_api.md]: type unneeded?

Comment (almostagile) [reply source/includes/_api.md]: Currently that is all that we support

Comment (srilq) [reply source/includes/_api.md]: Perhaps this makes it sound like the paginated response has a type, when it doesn't?

Comment (almostagile) [reply source/includes/_api.md]: Should perhaps say 
> _containing items_ of type ...

**Pull request merged**